lib/services/jira/jira_mapped_fields.rb

```diff
@@ -67,6 +74,19 @@ def aha_type_to_string(aha_type, aha_value)
       aha_value.to_s
     end
   end
+
+  def aha_type_to_user(aha_type, field)
+
+    if aha_type == "string"
+      user_resource.picker(field.value.strip).try(:[], :key)
+    else
+      field.email_value.each do |email|
+        potential_key = user_resource.picker(email).try(:[], :key)
+        return potential_key  unless potential_key.nil?
+      end
+      nil
+    end
+  end
```

Comment:
My only issue with doing this, is it doesn't follow how every other aha_type_to_X is structured. Consistency might be better here.